test/feature/test_loftr.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+import kornia.testing as utils  # test utils
+from kornia.feature import LoFTR
+from kornia.geometry import resize
+from kornia.testing import assert_close
+
+
+class TestLoFTR:
+    def test_pretrained_outdoor_smoke(self, device):
```

Comment:
@ducha-aiki ok, I guess here is where we should skip cuda version

Reply:
I haven't thought about such simple solution))) OK

Reply:
be aware of parsing issues with version as mentioned here too https://github.com/kornia/kornia/issues/1264